### ISSUE TITLE
libxml2-2.9.4 fails to compile on RHEL7 due to aclocal 1.15 requirement

### ISSUE THREAD

**Issue by akrherz** [Unidata/gempak#30]
This may be a red-herring, but will file this as well and attempt to track it down.

```
make[4]: Entering directory `/usr/local/gempak/extlibs/xml2/libxml2-2.9.4'
CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/akrherz/projects/gempak/extlibs/xml2/libxml2/missing aclocal-1.15 -I m4
/home/akrherz/projects/gempak/extlibs/xml2/libxml2/missing: line 81: aclocal-1.15: command not found
WARNING: 'aclocal-1.15' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make[4]: *** [aclocal.m4] Error 127
make[4]: Leaving directory `/usr/local/gempak/extlibs/xml2/libxml2-2.9.4'
```

**Comment by akrherz** [comment]
Ah, this may have something to do with file [timestamps when code is checked out from git](https://stackoverflow.com/a/33286344/1652109) , I just `autoreconf -f -i` from `libxml2` folder and things were happy.  Hmmmm

**Comment by akrherz** [comment]
`libxslt-1.1.29` suffers the same fate.  Unsure how best to fix this, maybe it is just a known one-off.

**Comment by mjames-upc** [comment]
has been fixed by c464d75, ready for 7.4.2. works for source builds on centos6 and 7 so far.
